CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- CASMTRIAGE-5846: Change `sysctl_set` to be dynamic to prevent failure when building images
+
+## [1.16.17] - 2023-08-16
+
+### Changed
+
```

Comment:
```suggestion
## [1.16.17] - 2023-08-16

### Changed

- CASMTRIAGE-5846: Change `sysctl_set` to be dynamic to prevent failure when building images
```